http-common/src/server.rs

```diff
@@ -113,7 +113,7 @@ macro_rules! make_service {
                                             } else {
                                                 let content_type = headers.get(hyper::header::CONTENT_TYPE).and_then(|value| value.to_str().ok());
 
-                                                match content_type.as_deref() {
+                                                match content_type.as_deref().and_then(|ct| ct.starts_with("application/json").then_some("application/json")) {
```

Comment:
This allows any string that starts with `"application/json"`, such as `"application/jsonn"`